### PR TITLE
feat: mock compile

### PR DESCRIPTION
**Pull request by rpalakkal**

### CODE REVIEW

File: circuit/js/src/scaffold.ts [hunk 1]
```diff
@@ -179,6 +206,12 @@ export abstract class AxiomBaseCircuitScaffold<T> extends BaseCircuitScaffold {
     return computeQuery;
   }
 
+  private getMockVk(): string[] {
+    const vkLen = 6 + 2 * DEFAULT_CIRCUIT_CONFIG.numAdvice + DEFAULT_CIRCUIT_CONFIG.numLookupAdvice;
+    const emptyVk = new Array(vkLen).fill(zeroHash);
```
Comment (yi-sun) [inline circuit/js/src/scaffold.ts:211]: Let's also do some form of `deadbeef`?

**Review by yi-sun — CHANGES_REQUESTED** [review]
Small changes and just confirming one thing on the usage of `mock` previously

File: circuit/js/src/cliHandler/compile.ts [hunk 2]
```diff
@@ -21,7 +22,7 @@ export const compile = async (
     const provider = getProvider(options.provider);
     const circuit = new AxiomBaseCircuit({
         f: f.circuit,
-        mock: true,
```
[the remark this answers is not in the document]
Comment (rpalakkal) [reply circuit/js/src/cliHandler/compile.ts:24]: not too sure, but doesn't seem like it was being used in `circuit.compile`. cc @ytham

[the remark this answers is not in the document]
Comment (ytham) [reply circuit/js/src/cliHandler/compile.ts:24]: it looks like it's only used to pass into AxiomSdkCore, which isn't used for circuit.compile

**Review by yi-sun — APPROVED** [review]

**Pull request merged**